RealmSwift/Tests/ListTests.swift

```diff
@@ -148,7 +148,7 @@ class ListTests: TestCase {
         XCTAssertEqual(str2, array[0])
         XCTAssertEqual(str1, array[1])
 
-        assertThrows(self.array[200])
+        assertThrows(self.array[200], named: nil)
```

Comment:
The vast majority of cases expect the name to be RLMException, so making that case the least verbose seems preferable to me. Maybe something like `assertThrowsWithoutCheckingName()`?